site/public/css/ta-grading.css

```diff
@@ -305,6 +305,11 @@ progress::-webkit-progress-value {
 
 #overall-comment-collapsed {
     padding: 0 0 0 20px;
+    max-height: 100px;
```

Comment:
1. Probably max-height is not working properly for textarea 
   ![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/42701709/55675954-27820480-58e8-11e9-89f0-1b7c2813ffbe.gif)
2. Should use javascript to break a word such that half-word-  and -last-half-word 
![ezgif com-video-to-gif (2)](https://user-images.githubusercontent.com/42701709/55676012-4fbe3300-58e9-11e9-8dee-7dc13c2f5d09.gif)

Reply:
Hey @drumilpatel2000 the max-height is working as intended - it keeps expanding till whatever size the edit mode requires. This is good for working on while editing the comment (observe how Facebook handles this). After saving the max height is 100px.

Reply:
Hey @xprilion Facebook increase height up to some limit and then scrollbar comes. This can be easily achieved by setting max-height.
Considering UI I does not consider this as better implementation as **Manual grading Total** goes out of the frame for long comments